packages/shared/src/interfaces/IWorkflowState.ts

```diff
@@ -0,0 +1,3 @@
+export interface IWorkflowState {
+  [key: string]: any;
```

Comment:
**suggestion** avoid using any. We are in a typed language, so lets take advantage of it ;)